src/main/java/duke/utils/FileStorage.java

```diff
@@ -0,0 +1,78 @@
+package duke.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+/**
+ * File storage object to handle storage in a local file
+ * Implements storage interface
+ */
+public class FileStorage implements Storage {
+
+    private String fileName;
+
+    /**
+     * File storage constructor
+     * @param fileName
+     */
+    public FileStorage(String fileName) {
+        this.fileName = fileName;
+    }
+
+    /**
+     * Create storage if it doesn't exist
+     * @throws IOException
+     */
+    public void createStorage() throws IOException {
+        //relative path
+        String relativePath = fileName;
+        File file = new File(relativePath);
+        if (file.createNewFile()) {
+            System.out.println(relativePath + " Save File Created");
+        } else System.out.println("File " + relativePath + " already exists");
+    }
+
+    /**
+     * Check if file that is to be created already exists
+     * @return
```

Comment:
It is always good to complete the Javadoc comments, in this case to specify what does the method returns.